README.md

```diff
@@ -188,8 +188,10 @@ If you are looking for the codebase of the current production version of GATK, p
     * Cloud tests require being logged into `gcloud` and authenticated with a project that has access
       to the test data.
 
-* To run just a single test class, run **`gradle test -Dtest.single=MyTestClass`**
-    * example: `gradle test -Dtest.single=ReadUtilsUnitTest`
+* To run a subset of tests, use gradle's test filtering (see [gradle doc](https://docs.gradle.org/current/userguide/java_plugin.html)), e.g.,
+    * `gradle test --tests *SomeSpecificTest`
```

Comment:
you might want to put `"` in your examples, otherwise my shell tries to expand the *.